### PR TITLE
Vb 2056 update api dummy out resp requ

### PR DESCRIPTION
**Pull request by AledWEvans**
added dummy API so UI guys can get on with work

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/VisitDto.kt [hunk 1]
```diff
@@ -60,9 +62,12 @@ data class VisitDto(
   @Schema(description = "The visit modified date and time", example = "2018-12-01T13:45:00", required = true)
   @field:NotBlank
   val modifiedTimestamp: LocalDateTime,
-
+  @Schema(description = "What was the last application method", required = true)
+  @field:NotNull
+  val lastApplicationMethod: ApplicationMethod,
```
Comment (AledWEvans) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/VisitDto.kt:67]: Do we need this @hutcheonb-moj? would rather not include it, if front dose not use it?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt [hunk 2]
```diff
@@ -271,7 +272,7 @@ class VisitService(
   @Deprecated("This method has been deprecated.")
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt:272]: is this method needed anymore?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt [hunk 3]
```diff
@@ -271,7 +272,7 @@ class VisitService(
...
   fun cancelVisit(reference: String, outcomeDto: OutcomeDto): VisitDto {
     LOG.debug("Enter legacy call to cancelVisit $reference")
-    val cancelVisitDto = CancelVisitDto(outcomeDto, NOT_KNOWN_NOMIS)
+    val cancelVisitDto = CancelVisitDto(outcomeDto, NOT_KNOWN_NOMIS, UNKNOWN)
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt:275]: instead of UNKONWN can ApplicationMethod be NOMIS here?

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt:275]: This is just to dummy out next ticket will put in correct value..

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request closed without merging**